lib/helpers/error-redirect.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
```

Comment:
Is there a test for this function? Or does this fork of the branch not have any tests setup?